### PR TITLE
Automated the setup part of the `WORKSHOP-README.md` into the `LAB-SETUP.ipynb` notebook

### PR DESCRIPTION
**Pull request by cedricvidal**
Automated the setup part of the `WORKSHOP-README.md` into the `LAB-SETUP.ipynb` notebook

Tested successfully end to end with a Skillable Tech Check environment

### CODE REVIEW

File: docs/workshop/LAB-MANUAL.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 
-**Username/email:** ++@lab.CloudPortalCredential(User1).Username++
-
-**Password:** ++@lab.CloudPortalCredential(User1).Password++
-
-**AzureEnvName:** ++AITOUR@lab.LabInstance.Id++
-
-**Subscription:** ++@lab.CloudSubscription.Id++
-
+```
```
Comment (marlenezw) [inline docs/workshop/LAB-MANUAL.md:3]: I'm not sure if this will render correctly in the lab manual. What is this change doing? @cedricvidal

Comment (marlenezw) [reply docs/workshop/LAB-MANUAL.md:3]: Actually, I just looked at the notebook and can understand what you were doing. Let me update this so it renders correctly and people could just copy the block and past in the cell!

Comment (cedricvidal) [reply docs/workshop/LAB-MANUAL.md:3]: Ok, feel free to commit the fix

File: docs/workshop/LAB-MANUAL.md [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
+# CREDENTIALS
+Username     = "++@lab.CloudPortalCredential(User1).Username++"
+Password     = "++@lab.CloudPortalCredential(User1).Password++"
+AzureEnvName = "++AITOUR@lab.LabInstance.Id++"
+Subscription = "++@lab.CloudSubscription.Id++"
+```
```
Comment (marlenezw) [inline docs/workshop/LAB-MANUAL.md:9]:
```suggestion
# CREDENTIALS

```
Username     = "@lab.CloudPortalCredential(User1).Username"

Password     = "@lab.CloudPortalCredential(User1).Password"

AzureEnvName = "AITOUR@lab.LabInstance.Id"

Subscription = "@lab.CloudSubscription.Id"
```
```

Comment (marlenezw) [reply docs/workshop/LAB-MANUAL.md:9]: The code suggestion above works in Skillable

File: docs/workshop/LAB-SETUP.ipynb [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+    "> **TIP**: You do not need to wait until the next script ends to start working on the first [workshop-1-intro.ipynb](workshop-1-intro.ipynb) notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! azd hooks run postprovision -e $AzureEnvName"
```
Comment (marlenezw) [inline docs/workshop/LAB-SETUP.ipynb:198]:
```suggestion
    "! echo 1 | azd hooks run postprovision -e $AzureEnvName"
```

**Review by marlenezw — APPROVED** [review]

**Comment by marlenezw** [comment]
LGTM, will merge and clean up some things on main!

**Pull request merged**